cdap-docs/developers-manual/source/building-blocks/datasets/system-custom.rst

```diff
@@ -101,7 +101,14 @@ To create a Dataset of type ``UniqueCountTable``, add the following into the App
 
 You can also pass ``DatasetProperties`` as a third parameter to the ``createDataset`` method.
 These properties will be used by embedded Datasets during creation and will be available via the
-``DatasetSpecification`` passed to the Dataset constructor.
+``DatasetSpecification`` passed to the Dataset constructor. For example, to create a 
```

Comment:
"dataset with multiple partitions" is very ambiguous unless you define it.  But here this term is confusing. "Table that is pre-split into multiple regions when the underlying storage is HBase" is more accurate, even though pretty complex.

Overall, this deserves separate section - may be in advanced topics space.